lib/pages/classes/model/class.dart

```diff
@@ -49,16 +50,36 @@ class ClassHeader {
   }
 }
 
+extension ClassExtension on ClassHeader {
+  Color get colorFromAcronym {
+    int r = 0, g = 0, b = 0;
+    if (acronym.isNotEmpty) {
+      b = acronym[0].codeUnitAt(0);
+      if (acronym.length >= 2) {
+        g = acronym[1].codeUnitAt(0);
+        if (acronym.length >= 3) {
+          r = acronym[2].codeUnitAt(0);
+        }
+      }
+    }
+    const int brightnessFactor = 2;
+    return Color.fromRGBO(
+        r * brightnessFactor, g * brightnessFactor, b * brightnessFactor, 1);
+  }
+}
```

Comment:
Move this to ClassIcon

This file should just define the data, not do anything database or UI-related